tests/test_dataset.py

```diff
@@ -311,6 +311,42 @@ def test_props_equal(self, constructor):
         assert list_props_uncached == list_props_cached
 
 
+@pytest.mark.parametrize(
+    "db",
+    [
+        "medium_db",
+    ],
+)
+def test_dataset_cache_root(tmpdir, request, db):
+    """Test configuration of cache root.
+
+    ``cache_root`` can be provided by different options,
+    in the following hierarchy:
```

Comment:
I changed the wording to use "precedence" instead of "hierarchy".

In the documentation it is briefly mentioned, see

![image](https://github.com/audeering/audbcards/assets/173624/77892a89-57be-4834-a62f-29c3259c8c95)

The `cache_root` variable is then discussed as part of `audbcards.Dataset`.